data/json/items/tools.json

```diff
@@ -9765,17 +9765,19 @@
     "color": "red",
     "name": "acetylene torch",
     "name_plural": "acetylene torches",
-    "description": "A compact tool kit intended for welding and cutting metal, this portable oxyacetylene torch includes a torch handle, cutting attachment, and two pre-filled gas cylinders all in an easy-to-carry tote.  In addition to its metalworking uses, you can activate it in order to destroy metal barriers.",
+    "description": "A compact tool kit intended for welding and cutting metal, this portable oxyacetylene torch includes a torch handle and cutting attachment in an easy-to-carry tote.  It requires connecting to pressurised cylinders of an appropriate welding gas before use.  In addition to its metalworking uses, you can activate it in order to destroy metal barriers.",
     "price": 20000,
     "material": ["steel", "plastic"],
-    "weight": 7600,
-    "volume": 16,
-    "max_charges": 240,
-    "initial_charges": 240,
+    "weight": 1600,
+    "volume": 4,
+    "max_charges": 0,
+    "initial_charges": 0,
     "charges_per_use": 4,
     "turns_per_charge": 0,
```

Comment:
This implementation does **not** yet support `turns_per_charge > 0`. A separate PR will address this and probably add an acetylene lamp running from a cylinder a demonstration item.